docs/static/dead-letter-queues.asciidoc

```diff
@@ -0,0 +1,149 @@
+[[dead-letter-queues]]
+=== Dead Letter Queues
+
+NOTE: The dead letter queue feature is currently supported for the
+<<plugins-outputs-elasticsearch>> output only. Support for additional outputs
+will be available in future releases of the Logstash plugins. Before configuring
+Logstash to use this feature, refer to the output plugin documentation to
+verify that the plugin supports the dead letter queue feature.
+
+By default, when Logstash encounters an event that it cannot process because the
+data contains a mapping error or some other issue, the Logstash pipeline 
+either hangs or drops the unsuccessful event. In order to protect against data
+loss in this situation, you can configure Logstash to write unsuccessful events
+to a dead letter queue instead of dropping them. 
+
+Each event written to the dead letter queue includes the original event along
+with metadata indicating when the event entered the queue. For example:
+
+//TODO: Need a better example here. 
+
+[source,json]
+-------------------------------------------------------------------------------
```

Comment:
yeah, no worries. I think it just ignored the `json` filter parsing because `original` field was not found :) All good now